nautobot/users/views.py

```diff
@@ -371,6 +407,20 @@ def create(self, request, *args, **kwargs):
             return redirect(self.get_return_url(request))
 
 
+class SavedViewClearView(GenericView):
```

Comment:
Should be possible to implement this as a custom action on `SavedViewUIViewSet` rather than a GenericView... in particular I'm concerned about permissions enforcement on a `GenericView`.

Reply:
Refactored to custom action